resources/translations/sv.edn

```diff
@@ -100,6 +100,7 @@
                    :disable "Avaktivera"
                    :disabled-and-archived-explanation "Att avaktivera ett föremål gör att det döljs för användarna. Att arkivera det gör att det även döljs från administrationsvyn."
                    :display-archived "Visa arkiverade"
+                   :display-order "Visa ordning"

```

Comment:
might also be "visningsordning" - google translate seems to understand "display order" as verb + subject instead here

Reply:
True, I was thinking too much of the previous translation that actually is in a place where an action is preferable.